js/lesson2/tutorial.md

```diff
@@ -304,6 +319,8 @@ To retrieve an item from the array, we use **square bracket notation**
 
 To get the first item `animals[0]`, the second `animals[1]` etc.
 
+> Create an array that includes 5 names and then access the 3rd item in the array? (don't forget to access the third item in the array you won't be doing [3])
```

Comment:
@KimberleyCook do you need the question mark at the end of this sentence? It's not a question but a statement.